package/opt/hassbian/suites/files/cloud9.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
```

Comment:
This service should be run after the network is available and most likely not after Home Assistant since a change to the config might cause Home Assistant not to start.

Reply:
Changed in next commit.